migrations/tenant/0029-create-prefixes.sql

```diff
@@ -48,6 +61,24 @@ DO $$
                 EXIT;
             ELSE
                 COMMIT;
+                PERFORM pg_sleep(delay);
+                -- Increase delay by 1 second for each iteration until 30
+                -- then reset it back to 1
+                SELECT CASE WHEN delay >= 30 THEN 1 ELSE delay + 1 END INTO delay;
```

Comment:
what's the rationale for increasing the delay by 1 second per iteration?
it looks like that'd lead to waves of different levels of iops

Reply:
for projects with a low number of objects, ~100k it will finish relatively quicker with an increasing delay. as opposed to taking a lot more time ex with a fixed 30s per iteration, min ~15mins